### DISCUSSION TITLE
128

### DISCUSSION THREAD

**Discussion by giscus** [CODe5753/giscus#10] [Comments]
# 128

프롤로그취업이 너무나도 어려운 시기에 개발자 취업을 꿈 꿔왔던 멘티가 있었습니다. 결국 치열한 노력 끝에 취업에 성공하여 기쁜 소식을 전해줬어요. 하지만 몇 개월 후, 제게 다시 연락을 해왔습니다. "힘들다, 퇴사하고 싶다"라는 지쳐있는 이야기를 말이죠. 저는 몇 가지 제 경험과 가치관에 대해 이야기를 해주었습니다. 대기업 공채 입사 이틀만에 퇴사를 결심한 이야기부터, 이직을 한 순간까지 어떻게 버텼는지, 그 과정에서 스스로 느낀 점들에 대해서요. 우리가 살아가는 길에는 많은 선택지가 있지만, 퇴사나 이직은 특히나 어렵고 고민되는 선택지 중 하나일 겁니다. 이 글은 퇴사라는 단어와 가까워지기 시작한, 혹은 아직은 멀게 느껴지지만 언젠가는 퇴사를 고민하게 될지도 모를, 4년차 이하의 개발자들을 위해 썼습..

https://imksh.com/128

**Comment by wjdrk70** [comment]
승현님은 글을 정말 잘 쓰십니다.
회피와 도망을 위한 퇴사를 하면 정말 나중이 힘들어지는게 맞는거 같습니다.
원하던 환경이 아니라면 오히려, 환경 변화를 위해 시도를 해보고 하얗게 불태웠을시 이직을 하는게 좋은 방안이라는 저의 주관적 생각입니다.
하지만 건강이 정말 안좋아졌을때는 퇴사를 하는게 맞습니다. 병가라도 내어서 몸 회복이 우선이니까요:)
그래서 항상 도망친곳에 낙원은 없다는 말을 되새기고는 하지만 생각만큼 쉽지는 않더군요
그래도 승현님 글을 보면 긍정적 에너지가 느껴져 오히려 으쌰으쌰하게되네요 좋은글 감사합니다!

**Reply by CODe5753** [reply]
@wjdrk70 안녕하세요.  
시간이 많이 지난 후에야 답글을 남기네요.  
칭찬 너무 감사드립니다 😀  
4개월이 지난 시점인데, 답글 알림으로 한번 더 긍정의 기운을 받아 가시길 바랍니다!! ㅎㅎ

**Comment by kimkyury** [comment]
오랜만이예요, 나아가고 싶은데, 도망치는 것 같고.. 다만 마음만 떠나가고 그랬는데 어느 방향으로든 정리가 되었어요. 감사합니다.

**Reply by CODe5753** [reply]
@kimkyury  
오!! 너무 오랜만이네~~  
항상 모범생의 정석을 보여줬어서 대단하다고 생각했는데,  
지금 겪는 어려움과 정답이 없는 선택들도 나중엔 다 밑거름이고 좀 더 현명한 선택을 하는 데 도움이 될 거니까   
늘 잘해왔듯이 앞으로도 잘 할거라 믿는다!!  
항상 응원할게 화이팅!!!! 🔥
